assemblies/data-access-plugin/src/main/resources/resources/test/query-sample.html

```diff
@@ -13,6 +13,7 @@
 </script>
 <script language="javascript" src="../web/dojo/dojo.js" type="text/javascript"></script>
 <script language="javascript" src="../web/messages/Messages.js" type="text/javascript"></script>
+<script language="javascript" src="../web/pentaho-csrf.js" type="text/javascript"></script>
 <script language="javascript" src="../web/pentaho-ajax.js" type="text/javascript"></script>
```

Comment:
Only need the `pentaho-ajax` include, if it then loads common-ui's pentaho-ajax.